commands/utility/remindme.js

```diff
@@ -0,0 +1,30 @@
+exports.run = async (handler, message, args, pre) => {
+	if (!args.hours && !args.minutes && !args.seconds) {
+		return message.channel.send('No arguments given.')
+	}
+
+	var time = ((args.hours ? args.hours : 0) * 3600000) + ((args.minutes ? args.minutes : 0) * 60000) + ((args.seconds ? args.seconds : 0) * 1000)
+
+	message.channel.send(`Set timer for ${args.hours ? `${args.hours} hours, ` : ''}${args.minutes ? `${args.minutes} minutes, ` : ''}${args.seconds ? `${args.seconds} seconds ` : ''}from now${args.reason ? ` for \`${args.reason}\`` : ''}.`)
```

Comment:
This line needs to be `await`ed, otherwise an error here will lead to an UnhandledPromiseRejectionWarning.